5.0/en/0x12-V3-Session-management.md

```diff
@@ -16,7 +16,7 @@ As previously noted, these requirements have been adapted to be a compliant subs
 | # | Description | L1 | L2 | L3 | CWE | [NIST &sect;](https://pages.nist.gov/800-63-3/sp800-63b.html) |
 | :---: | :--- | :---: | :---: | :---: | :---: | :---: |
 | **3.1.1** | Verify that the application never reveals session tokens in URL parameters. | ✓ | ✓ | ✓ | 598 | |
-| **3.1.1** | Verify that the application performs all session tokens verification using a trusted, back-end service. | ✓ | ✓ | ✓ | 603 | |
+| **3.1.1** | [ADDED] Verify that the application performs all session tokens verification using a trusted, back-end service. | ✓ | ✓ | ✓ | 603 | |
```

Comment:
there seems to be some mistake.

If we "replace" requirement, it makes sense to mark one requirement [DELETED, + reason] and new requirement must be with new unique identifier with [ADDED] label

Reply:
OK, does this look better now @elarlang:
https://github.com/OWASP/ASVS/pull/1217/commits/d4f097d47fb1c7fce7bf5df3e9196d30527a94f5
https://github.com/OWASP/ASVS/pull/1217/files